Project 12/src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,36 @@
+import calculator.ConsoleCalculator;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CalculatorTest {
+    @Test
+    public void testCalculatorAdd() {
+        int resultAdd = ConsoleCalculator.operation(6, "+", 7);
+        Assert.assertEquals(resultAdd, 13);
+
+    }
+
+    @Test
+    public void testCalculatorSubtract() {
+        int resultSubtract = ConsoleCalculator.operation(17, "-", 10);
+        Assert.assertEquals(resultSubtract, 7);
+    }
+
+    @Test
+    public void testCalculatorDivide() {
+        int resultDivide = ConsoleCalculator.operation(10, "/", 2);
+        Assert.assertEquals(resultDivide, 5);
+    }
+
+    @Test
+    public void testCalculatorDuplicate() {
+        int resultDuplicate = ConsoleCalculator.operation(10, "*", 2);
+        Assert.assertEquals(resultDuplicate, 20);
+    }
+
+    @Test(expectedExceptions = ArithmeticException.class)
+    public void testException() {
+        int resultDivideZero = ConsoleCalculator.operation(5, "/", 0);
```

Comment:
you don't need a variable resultDivideZero, because you don't use it after you assign it

Reply:
I don`t get it, and where will be the test if I delete it? Will it be just like that ? @Test(expectedExceptions = ArithmeticException.class)
    public void testException() {

Reply:
Just call the method:
@test(expectedExceptions = ArithmeticException.class)
public void testException() {
        ConsoleCalculator.operation(5, "/", 0);
}

Reply:
ok thanks)